src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -518,4 +520,38 @@ def remove_file_post
       end
     end
   end
+
+  describe 'GET #trigger_services' do
+    before do
+      login user
+    end
+
+    context 'with right params' do
+      let(:post_url) { "http://localhost:3200/source/#{source_project}/#{service_package}?cmd=runservice&user=#{user}" }
+
+      before do
+        get :trigger_services, params: { project: source_project, package: service_package }
+      end
+
+      it { expect(a_request(:post, post_url)).to have_been_made.once }
+      it { expect(flash[:notice]).to eq('Services successfully triggered') }
+      it { is_expected.to redirect_to(action: :show, project: source_project, package: service_package) }
+    end
+
+    context "without a service file in the package" do
+      let(:post_url) { "http://localhost:3200/source/#{source_project}/#{source_package}?cmd=runservice&user=#{user}" }
+      let(:error) do
+        "Services couldn't be triggered: <status code=\"404\">\n  <summary>no source service defined!</summary>" \
+        "\n  <details>404 no source service defined!</details>\n</status>\n"
+      end
```

Comment:
and what is the alternative? If I keep it in one line it is too long and Rubocop complains.